src/service/fixed-layer.js

```diff
@@ -607,8 +607,10 @@ export class FixedLayer {
     if (!this.transfer_ || this.fixedLayer_) {
       return this.fixedLayer_;
     }
-    this.fixedLayer_ = this.ampdoc.win.document.createElement('body');
-    this.fixedLayer_.id = '-amp-fixedlayer';
+    const doc = this.ampdoc.win.document;
+    const bodyId = doc.body.id;
+    this.fixedLayer_ = doc.createElement('body');
```

Comment:
But we'll have to reset `fixedLayer.style` because we override it for real body for many things.